### DISCUSSION TITLE
v3.1.0

### DISCUSSION THREAD

**Discussion by mrghozzi** [mrghozzi/myads#53] [Announcements]
**Improvement**: Site directory
**Update**: privacy policy
**Fix**: Error in advertisement manager template
**Improvement**: admin_nav & admin_home template
**Addition**: Follow all posts or posts from those you follow in the community
**Improvement**: Private message template
**Improvement**: Community posts and images template
**Fix**: Bugs and vulnerabilities
**Addition**: The ability to arrange forum categories
**Added**: Description of forum categories

https://github.com/mrghozzi/myads/releases/tag/v3.1.0

**Comment by alaa-eg** [comment]
يرجى اضافة ملف robots.txt في التحديث القادم

حاولت اضافته ولكنه لا يعمل ، ويتم توجيه الرابط للصفحة الرئيسية

هناك مشكلة في أرشفة /templates

مثال
/templates/_panel/status/post_comment.php?s_type=1&tid=665&limet=5
/templates/_panel/status/post_comment.php?s_type=100&tid=204&limet=5

**Reply by mrghozzi** [reply]
https://github.com/mrghozzi/myads/issues/65

**Comment by morad19831983** [comment]
Hello, I have a problem  https://

sll  It does not work even though the hosting is there  sll
![2024-04-06_01-29-16](https://github.com/mrghozzi/myads/assets/166183366/4ec5ab4d-f1b8-46a3-9029-591a710ac7c2)

**Reply by mrghozzi** [reply]
Welcome.
Make sure that you have set up your SSL certificate correctly or that you are not misusing the domain name.
The script has nothing to do with the SSL certificate.
There is more than one site that uses the script and the phrase (the site is not safe for them) does not appear.
This is an example https://adstn.is-best.net

**Comment by morad19831983** [comment]
The certificate has been activated. Thank you. Can I change the template from left to right?

**Reply by mrghozzi** [reply]
We are working on this in upcoming updates
https://github.com/mrghozzi/myads/issues/38

**Comment by morad19831983** [comment]
In the next release, we hope to introduce a feature to purchase points
